src/components/Tabs/Tabs.jsx

```diff
@@ -8,12 +8,12 @@ import style from './style.scss';
 
 type Tab = {|
   label: string,
-  title: Node,
+  title: React$Node,
 |};
 
 type Props = {|
   items: Array<Tab>,
-  onChange?: (Node, number) => void,
+  onChange?: (string, number) => void,
```

Comment:
Tab title is a `React$Node`, though, no?

Reply:
Nope

Reply:
Well maybe, but if you look at the code this function gets passed a string